src/site/includes/header.html

```diff
@@ -33,27 +34,31 @@
 <meta name="msapplication-TileColor" content="#144073">
 
 <!-- CSS -->
-<link rel="stylesheet" href="/generated/style.css">
-{% if entryname %}
-<link rel="stylesheet" href="/generated/{{ entryname }}.css">
+<link rel="stylesheet"
+  data-entry-name="style.css"
+  href="/generated/style.css">
 
-{% else %}
-<link rel="stylesheet" href="/generated/static-pages.css">
-{% endif %}
+<link rel="stylesheet"
+  data-entry-name="{{ entryname | default: 'static-pages' }}.css"
+  href="/generated/{{ entryname | default: 'static-pages' }}.css">
 
 <!-- Old IE -->
 
 <!--[if lt IE 9]>
 <link rel='stylesheet' href='/assets/css/ie.css'>
 <![endif]-->
 
-<script defer nomodule src="/generated/polyfills.entry.js"></script>
-<script defer src="/generated/vendor.entry.js"></script>
-{% if entryname %}
-<script defer src="/generated/{{ entryname }}.entry.js"></script>
-{% else %}
-<script defer src="/generated/static-pages.entry.js"></script>
-{% endif %}
+<script defer nomodule
+  data-entry-name="polyfills.js"
+  src="/generated/polyfills.entry.js"></script>
+
+<script defer
+  data-entry-name="vendor.js"
+  src="/generated/vendor.entry.js"></script>
+
+<script defer
+  data-entry-name="{{ entryname | default: 'static-pages' }}.js"
+  src="/generated/{{ entryname | default: 'static-pages' }}.entry.js"></script>
```

Comment:
This PR adds that `data-entry-name` attribute to our Webpack references. This way, we can quickly filter our DOM in our `addAssetHashes` build step.

Reply:
The values of `data-entry-names` align exactly with the keys in the `generated/file-manifest.json`.

Reply:
These seem like good inline comments.

Reply:
+1 for documentation somehow, but I'm not sure if inline comments are the best choice (annoyingly). I'm pretty sure they'll show up in production, which I think we probably don't want.

Reply:
I renamed the plugin to `process-entry-names` and changed it so that we no longer need `src` or `href` values at all for Webpack assets. I think this should make it clearer that the `data-entry-name` attribute is meaningful.

Reply:
I really don't think there's much value in a Liquid comment, but I can do that if you feel strongly.